SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -805,7 +807,7 @@ private boolean checkCampaignRules(int elementCount, String elementName, String
     @SuppressLint("UseSparseArrays")
     protected SwrveConversation _getConversationForEvent(String event) {
         SwrveConversation result = null;
-        SwrveCampaign campaign = null;
+        SwrveMessageCampaign campaign = null;
```

Comment:
That variable isn't used for anything anymore if I am correct, so could be removed